test/Microsoft.Azure.WebJobs.Host.UnitTests/PublicSurfaceTests.cs

```diff
@@ -16,6 +14,7 @@ namespace Microsoft.Azure.WebJobs.Host.UnitTests
     /// fail any time new dependencies or public surface area are added, ensuring
     /// we review such additions carefully.
     /// </summary>
+    [Trait(TestTraits.CategoryTraitName, TestTraits.ScaleMonitoring)]
```

Comment:
Remove or add another category name.

Reply:
I added this here because when doing scaler work, we're changing the surface area and want to run these tests. Note that tests can have multiple traits so this isn't a problem.